enhancements/proposed/20200921-dedicated-network-kic/dedicated-network-kic.md

```diff
@@ -0,0 +1,40 @@
+# Static IP for minikube containers KIC Drivers (docker/podman)
+
+design by @medyagh
+
+## Prototype [PR 9294](https://github.com/kubernetes/minikube/pull/9294):
+
+Each minikube cluster will get it is own network, instead of using docker’s owned default network which is unmodifiable by the user.
+
+on “minikube start”, minikube will try to create a docker network at a default subnet
```

Comment:
Can you describe how the default subnet was chosen, and what occurs if that default subnet happens to be the one the host is using?